### PR TITLE
📚DOCS: Add pvlib python PVSC tutorial to gallery

### PR DESCRIPTION
**Pull request by mikofski**
FYI: @kanderso-nrel , @shirubana , @wholmgren , @cwhanse

### CODE REVIEW

File: docs/gallery.yml [hunk 1]
```diff
@@ -173,3 +173,7 @@
   repository: https://github.com/euroargodev/argoonlineschool
   image: https://euroargodev.github.io/argoonlineschool/_static/logoAoS.png
   website: https://euroargodev.github.io/argoonlineschool
+- name: "PVSC pvlib python Tutorial"
+  repository: https://github.com/PVSC-Python-Tutorials/PVSC48-Python-Tutorial
+  image: https://github.com/pvlib/pvlib-python/blob/master/docs/sphinx/source/_images/pvlib_logo_vert.png
```
Comment (kanderso-nrel) [inline docs/gallery.yml:178]:
```suggestion
  image: https://raw.githubusercontent.com/pvlib/pvlib-python/master/docs/sphinx/source/_images/pvlib_logo_vert.png
```

Maybe?

Comment (mikofski) [reply docs/gallery.yml]: yep, working now: 
image

**Comment by mikofski** [comment]
@maintainers thank you for your time and effort put into creating this amazing open source project! We appreciate you. Can you please consider our PR? thanks!

**Comment by mikofski** [comment]
Hi Jupyterbook maintainers, I think this PR is ready to reviewed. Can someone please take a look? Thank you very much! ❤️

**Comment by mikofski** [comment]
@choldgraf I apologize for bothering you , but just want to make sure I'm not missing anything. Do you know if I should tag someone to review this, or you can let me know if I just need to be more patient. Sorry. thanks!

**Comment by chrisjsewell** [comment]
All good cheers

**Pull request merged**